go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/aikido/sc-agent
+
+go 1.21
```

Comment:
Probably want to use a newer version of Go, 1.21 has been unsupported for a while.